lib/feature/pokemon_details_page.dart/widgets/evolution_tab/widgets/pokemon_evolution_tab_widget.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex_async_redux/api/models/species_model.dart';
+import 'package:pokedex_async_redux/utilities/constants.dart';
+import 'package:pokedex_async_redux/utilities/extensions.dart';
+
+class PokemonEvolutionWidget extends StatelessWidget {
+  const PokemonEvolutionWidget({
+    required this.thisSpecies,
+    required this.arrowCheck,
+  });
+  final List<Species> thisSpecies;
+  final bool arrowCheck;
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: thisSpecies
+          .map(
+            (thisSpecies) => Container(
+              margin: arrowCheck ? EdgeInsets.fromLTRB(30, 0, 0, 0) : EdgeInsets.all(0),
+              height: 80,
+              width: 80,
+              child: Stack(
+                clipBehavior: Clip.none,
+                children: [
+                  if (arrowCheck)
+                    Positioned(
+                      left: -55,
+                      child: Container(
+                        width: 50,
+                        height: 50,
+                        child: Icon(Icons.arrow_right_outlined),
+                      ),
+                    ),
+                  Image.network(
+                    '$pokemonImgUrl${thisSpecies.id ?? ''}.png',
+                    width: 50,
+                    height: 50,
+                  ),
+                  Positioned(
```

Comment:
try adjusting your simulator's font size, The texts overlap each other. And the arrows aren't centred anymore. Better to just have a simple column here than to use stack. For the arrows just put it in your evolution tab. so it's always at the centre of two pokemons
![Screenshot 2022-08-01 at 5 32 02 PM](https://user-images.githubusercontent.com/47196695/182119196-a1de5150-563a-4934-be69-424924b2fd8a.png)